### DISCUSSION TITLE
Pepperplate import?

### DISCUSSION THREAD

**Discussion by yourfavoriteweapon** [TandoorRecipes/recipes#525] [General]
I'm not able to get pepperplate to import properly.  its in the documentation but i don't see it as available in the dropdown list. i tried using default, but it doesn't work.  Anyone have success with that import?

**Comment by vabene1111** [comment]
have you updated to the latest version that was released yesterday ? (0.15.0)

**Reply by yourfavoriteweapon** [reply]
I am using it thru the unraid docker, it doesn't show an update availalble.  but i am on 0.14.5 at the moment.

**Reply by vabene1111** [reply]
that is interesting, i thought unraid just updates automatically updates trough docker .. maybe it takes some time.
If not the unraid image is maintained by the community so it is probably going to be updated soon.

Whatever is the case you will definitly need v0.15.0 to use the pepperplate import, i guess it will most likely be there in a few days, if not let me know and i will check
